wp-meta.test/public_html/style.css

```diff
@@ -1,13 +1,34 @@
+body {
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+    background-color: #2b303b;
+    color: #c0c5ce;
+    padding:1em;
+    font-size:20px;
+    line-height: 1.4em;
+    max-width: 35em;
+    margin: 1em auto;
+}
 a {
-	color: #21759B;
-	text-decoration: none;
+    color: #ebcb8b;
+    text-decoration: none;
+}
+a:hover,
+a:active {
+    text-decoration: underline;
+}
+code,
+pre,
+h1,
+strong {
+    color: #eff1f5;
+}
+h1 {
+    margin-top: 2em;
 }
-
-a:hover {
-	color: #0F3647;
+#wp_logo {
+    text-align: center;
 }
-
-code {
-	padding: 3px 5px 1px 5px;
-	background-color: #e2e2e2;
+#wp_logo img {
+    width: 6em;
+    height: 6em;
```

Comment:
This kind of thing feels like it should be a separate PR from stuff related to broken provisioning steps.

Reply:
That page has broken styling, this brings a major readability fix, I'm intending to circle back and do a proper update with a back to dashboard and a WP logo